server_admin/topics/identity-broker/mappers.adoc

```diff
@@ -22,6 +22,11 @@ image:{project_images}/identity-provider-mapper.png[]
 Select a mapper from the `Mapper Type` list.  Hover over the tooltip to see a description of what the mapper does.  The
 tooltips also describe what configuration information you need to enter. Click `Save` and your new mapper will be added.
 
+The mapper will update user information when the user logs in repeatedly according to the `Sync Mode Override`. Choose `inherit` to 
```

Comment:
My vote would be to make the documentation a bit clearer for someone, who doesn't know so much context. I would change the last few sentences to something like:

_Choose `legacy` to
keep the mapper behaviour to keep the behaviour in the old Keycloak versions. Choose `import` to only import data when the user is first created in Keycloak during the first login to Keycloak with particular identity provider. Choose `force` to always update user data during every user login._

Maybe the itemized list will be even better :) Also the word "Keycloak" will be good to replace with "{project_name}" (See other point for more details).

Reply:
Choose legacy to use the behavior in the previous {project_name} version. Choose import to import only data from when the user was first created in {project_name} during the first login to {project_name} with a particular identity provider. Choose force to update user data at each user login.